File: Unidad 3-Funciones/Ejercicio42.js
/*
Autor: Jose Eduardo Mendez Verdejo  16/Febrero/19
Entradas: Numero
Salidas: EL numero y su raiz
Procedimiento: Calcular la raiz de cada uno de los 100 primeros numeros naturales
*/

var numero = 1;
var raiz;
while(numero <= 100){

	raiz = proceso(numero);
	salida(numero,raiz);
	numero++; //se incrementa en 1 el numero
	}




function proceso(numero){
	var raiz;
	raiz = Math.sqrt(numero); // se calcula la raiz del numero
	return raiz;
}

function salida(numero,raiz){
	alert("Numero: " + numero + "Raiz: " + raiz);
	
}

File: Unidad 3-Funciones/Ejercicio42.py
#Ejercicio #42
#Escribe el programa que calcule las raíces cuadradas de los cien primeros números naturales. Como resultado se desean dos columnas de valores: #
#En la primera columna se escribirán los números naturales y en la segunda sus raíces correspondientes.
#Pamela Citlali Canul Chacón (Equipo 'about:blank')

#El problema no necesita entradas 

#Proceso
import math 
#Se define la funcion
def raiz (numRaiz,num):
    #Se obtiene la raiz
    for num in range (1,101):
        numRaiz=math.sqrt(num)
        print (num)
        print (numRaiz)
#Salida 
print (raiz (numRaiz=0,num=0))